gdc_lib_main/functions/util/fn_chooseSpawnPos.sqf

```diff
@@ -30,52 +43,79 @@ if ((count _units) == 0) then {
 
 // si le joueur a le rang requis il peut déplacer le marqeur avec un clic gauche sur la carte
 if (rank player == _rank) then {
-	[_mk,_blist,_water] onMapSingleClick {
+	[_mk, _blist, _wlist, _water] onMapSingleClick {
+		params ["_mk", "_blist", "_wlist", "_water"];
+
 		_valid = true;
+		// Check if whilisted area
+		// If no whitelist marker, it is valide because all map is valid
+		if(!count _wlist == 0) then {
```

Comment:
wrong position for ! operator. Must be !( and not (!

Reply:
Must be
> if(count _wlist == 0) then {

The comment is clear, "NO marker" so == 0